packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -704,6 +776,17 @@ class TransformLayer extends OffsetLayer {
     builder.pop();
   }
 
+  @override
+  S find<S>(Offset regionOffset) {
+    if (_invertedTransform == null) {
+      _invertedTransform ??= new Matrix4.zero();
+      _transform?.copyInto(_invertedTransform)?.invert();
```

Comment:
there's a method or constructor or something that'll get you a new matrix that represents the inverse of another, which'll save you having to manually create the new one.

Reply:
Fixed